test/unittests/test_event_scheduler.py

```
@@ -16,92 +17,124 @@
 
         es.check_state()
-        self.assertEqual(emitter.emit.call_args[0][0].msg_type, 'test')
+        self.assertEqual(emitter.emit.call_args[0][0].msg_type, "test")
         self.assertEqual(emitter.emit.call_args[0][0].data, {})
         es.shutdown()
 
+    @patch("threading.Thread")
+    @patch("json.load")
+    @patch("json.dump")
+    @patch("builtins.open")
+    def test_list_events_handler(self, mock_open, mock_dump, mock_load, mock_thread):
```

Ruff (0.13.3)

Unused method argument: `mock_dump`

(ARG002)